src/main/scala/no/ndla/conceptapi/service/ConverterService.scala

```diff
@@ -168,6 +176,12 @@ trait ConverterService {
         case _                      => None
       }
 
+      val newMetaImage = concept.metaImage match {
+        case Right(Some(_)) =>
+          concept.metaImage.map(m => domain.ConceptMetaImage(m.get.id, m.get.alt, lang)).toSeq
+        case _ => Seq.empty
+      }
```

Comment:
Samme med `.get` her:
Gjør noe sånt:
```suggestion
      val newMetaImage = concept.metaImage.toSeq
        .flatMap(meta => meta.map(m => domain.ConceptMetaImage(m.id, m.alt, lang)))
```
Eller sånt: :smile: 
```suggestion
      val newMetaImage = concept.metaImage match {
        case Right(meta) => meta.map(m => domain.ConceptMetaImage(m.id, m.alt, lang)).toSeq
        case Left(_)     => Seq.empty
      }
```